packages/camera/camera_android_camerax/lib/src/android_camera_camerax.dart

```diff
@@ -10,7 +10,7 @@ import 'package:camera_platform_interface/camera_platform_interface.dart';
 import 'package:flutter/services.dart'
     show DeviceOrientation, PlatformException;
 import 'package:flutter/widgets.dart'
-    show Size, Texture, Widget, visibleForTesting;
+    show RotatedBox, Size, Texture, Widget, visibleForTesting;
```

Comment:
I used `RotatedBox` to rotate the preview mostly because that is what is used in the plugin code, but if there is a better way to rotate it, let me know.